build/shadowsocks_config.js

```diff
@@ -23,8 +23,8 @@ var __extends = (this && this.__extends) || (function () {
     };
```

Comment:
Why do we still generate this file? Can we get rid of this file? We should depend on the ts code instead.
It's very confusing. It can go out of sync and it's not clear what will actually be used.

Reply:
Just to close out this thread- I looked into this and it's probably possible, but requires making a bunch of tricky config changes on both the ssconfig and server/client side (the client imports are especially tricky because of the way we use babel-loader.)

https://github.com/Microsoft/TypeScript/issues/12358 useful summary of the issue:

> We got everything working for us but it became very obvious from reactions to this issue that it was not really a use case that was being considered and that *we would always be fighting to tooling to achieve what we wanted.*

So I think we should stick with generating the js/d.ts exports in the standard way.